public/blog.html

```diff
@@ -0,0 +1,88 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Blog</title>
+    <link rel="stylesheet" href="/app.css">
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.css">
+  </head>
+  <body>
+    <div class="full-page">
+    <nav class="top-nav">
+      <div class="top-nav__container">
```

Comment:
For the top-nav, you can add one more level with your existing `.container` styles to make sure that the navbar lines up with the various grid contents.

![screen shot 2016-10-18 at 1 32 27 pm](https://cloud.githubusercontent.com/assets/2532004/19491206/b1a8f46a-9537-11e6-9bdd-86be75d6565a.png)